legend-engine-xt-relationalStore-memsql-execution-tests/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/memsql/test/MemSQLTestContainerWrapper.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.plan.execution.stores.relational.connection.memsql.test;
+
+public class MemSQLTestContainerWrapper
```

Comment:
What is the intended purpose of the class?  To me seems little redundant.

Reply:
Just a cleaner interface for test usage. It mirrors some of the others that exist already...once I use in actual tests, we can reassess the use I think?